src/utils/getRequiredSignoffs.js

```diff
@@ -0,0 +1,35 @@
+import { OBJECT_NAMES } from './constants';
+import RequiredSignoffs from '../views/RequiredSignoffs/ListSignoffs/RequiredSignoffs';
```

Comment:
It looks like the object being imported here is more of a data model than a view, is it normal for those to live in `views` in the React world? I'll totally defer to you on this, I just want to make sure I understand the architecture.

Reply:
Makes sense. I'll move it to be under `views/RequiredSignoffs`.

Reply:
Done.